apps/src/code-studio/components/aiTutor/validationTutor.jsx

```diff
@@ -52,15 +59,15 @@ const ValidationTutor = ({levelId}) => {
         </h4>
       )}
       {validationPassed && <h4>🎉 Your tests are passing. Wahoo!</h4>}
-      {hasRunOrTestedCode && !hasCompilationError && !validationPassed && (
+      {enableAskAITutor && (
         <>
           <h4>Why aren't my tests passing?</h4>
           <Button
             text="Ask AI Tutor"
             isPending={isWaitingForAIResponse}
             pendingText="waiting"
             onClick={() => handleSend(studentCode)}
-            disabled={true}
+            disabled={enableAskAITutor}
```

Comment:
@molly-moen This add more requirements to when the button can be enabled, which I think addresses what you noticed in the video.